src/c-stdaux.h

```diff
@@ -675,6 +675,23 @@ static inline void *c_memcpy(void *dst, const void *src, size_t n) {
         return dst;
 }
 
+/**
+ * c_memcmp() - Compare memory area
+ * @s1:        Pointer to one area
+ * @s2:        Pointer to other area
+ * @n:          Length of area to compare
+ *
+ * Compare the memory of size ``n`` of ``s1`` and ``s2``, just as ``memcmp(3)``
+ * does, except this function allows either to be ``NULL`` if ``n`` is zero.
+ *
+ * Return: the comparison result for ordering.
+ */
+static inline int c_memcmp(void *s1, const void *s2, size_t n) {
```

Comment:
I think you forgot the `const` in `s1` (copy-pasted from `c_memcpy`? :)) I fixed it.